packages/cardano-services/src/NetworkInfo/openApi.json

```
@@ -173,6 +254,137 @@
             "$ref": "#/components/schemas/LovelaceSupply"
           }
         }
+      },
+      "LedgerTipResponse": {
+        "type": "object",
+        "required": [
+          "blockNo",
+          "slot",
+          "hash"
+        ],
+        "properties": {
+          "blockNo": {
+            "type": "number",
+            "example": 3274726
+          },
+          "slot": {
+            "type": "number",
+            "example": 49032415
+          },
+          "hash": {
+            "type": "string",
+            "example": "5caede44f4a5a775443095159cd42c8a64f35494086957ab3e04624015a6e13c"
+          }
+        }
+      },
+      "WalletProtocolParamsResponse": {
```

```suggestion
      "CurrentWalletProtocolParametersResponse": {
```
Any exposed API should used the complete name

---

de39caa57407d90f0623275fc237cb8737040a13